### ISSUE TITLE
Switch to using accessible colours - a darker purple [Feature Request]

### ISSUE THREAD

**Issue by lukew-cogapp** [Laravel-Backpack/CRUD#4875]
Hello,

We're using Backpack on a project but have noticed that the purple you use is not accessible on a white background (nor is the blue alternative).

They both fail to meet WCAG AA colour contrast ratio. Whilst we're aware that we can create overrides for these, it would be nice for the wider world (and especially those with any visual impairments) to have this available by default.

Alternatively, a simple-to-implement "primary colour" configuration variable would be great, so that we could change this to something accessible without needing to go through the _full_ custom config route.

Thank you.

Pasted image from https://webaim.org/resources/contrastchecker/:
![image](https://user-images.githubusercontent.com/83020016/210383307-47ba64b0-b6b9-4721-84df-30cccc0f4798.png)
